de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/impexp/AbstractImportProcess.java

```diff
@@ -433,7 +433,7 @@ protected void afterImport()
 	protected final void runSQLAfterRowImport(@NonNull final ImportRecordType importRecord)
 	{
 		final List<DBFunction> functions = getDbFunctions().getAvailableAfterRowFunctions();
-		final Optional<Integer> dataImportId = InterfaceWrapperHelper.getValueOrNull(importRecord, COLUMNNAME_C_DataImport_ID);
+		final Optional<Integer> dataImportId =  Optional.ofNullable(InterfaceWrapperHelper.getValueOrNull(importRecord, COLUMNNAME_C_DataImport_ID));
```

Comment:
```suggestion
		final Optional<Integer> dataImportId =  InterfaceWrapperHelper.getValue(importRecord, COLUMNNAME_C_DataImport_ID);
```

Reply:
@metas-ts , no is not ok; InterfaceWrapperHelper.getValue will throw an exception when the column is not found. And I want to return null if the column is not found.